db/migrate/20230119024431_add_user_id_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ] I think you should make use of Author model which is an alias to the User model and is the actual foreign key (AuthorId)
add_references :likes :author, foreign_key: { to_table: 'users' }